benchmarks/benchmark.ml

```diff
@@ -1,6 +1,53 @@
 open Core.Std
 open Core_bench.Std
 
+module Http = struct
+  open Re
+
+  let group re = function
+    | true -> group re
+    | false -> re
```

Comment:
Don't bother with that: build everything with groups and add `Re.no_group` at the end if you want to remove them.